realm/src/main/java/io/realm/BaseRealm.java

```diff
@@ -77,6 +88,7 @@
     protected SharedGroupManager sharedGroupManager;
     protected boolean autoRefresh;
     Handler handler;
+    ColumnIndices columnIndices;
```

Comment:
I read entire code again and I understood `columnIndices` is not used from Dynamic APIs.

It's better to move `columnIndices` to Realm.java in order not to accidentally access this field from Dynamic APIs.